solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/indexing/DocumentIndexingPackage.java

```diff
@@ -205,25 +207,27 @@ public void setChildren(List<PID> children) {
 		this.children = children;
 	}
 
-	public Map<String, List<String>> getTriples() throws IndexingException {
+	public Model getTriples(String path) throws IndexingException {
 		if (triples == null) {
-			triples = loader.loadTriples(this);
+			triples = loader.getRepository().getContentObject(path).getModel();
```

Comment:
This is going around the cache, which we don't want to do.  Instead, you can call `getContentObject()` from this same class to get the content object.

To be honest though, this is kind of tangential to the current PR.  I'm working on SetAccessControlFilter for another ticket (the only non-deprecated place this is used), and the new implementation isn't using getFirstTriple anymore since that logic is happening in the ACL classes.  So this change is fine, I'd just recommend not changing things you don't need immediately in case of situations like this :)